packages/liferay-js-publish/src/index.js

```diff
@@ -175,11 +171,53 @@ function getRemote() {
 	throw new Error('Unable to determine remote repository URL');
 }
 
+function isPrereleaseVersion(pkg) {
+	const {version} = pkg;
+
+	const parts = version.split('.');
+
+	// De facto standard for prerelease versions is; 10.0.0-alpha.1
+
+	if (parts.length > 3) {
+		return true;
+	}
+
+	// look for any non-number character
+	if (parts[2].indexOf('-') !== -1) {
```

Comment:
Please change to `includes('-')`

More readable than `indexOf('-') !== 1`

Reply:
:man_facepalming: I always write `contains` because of Java, then look in MDN and see that it doesn't exist and keep falling back to `indexOf` while continuously forgetting `includes`. Let's see if this is the last time :pray: